lib/pentagon/__init__.py

```diff
@@ -59,6 +63,13 @@ class PentagonProject():
     _production_kubernetes_v_log_level = '<production_kubernetes_v_log_level>'
     _production_kubernetes_network_cidr = '<production_kubernetes_network_cidr>'
 
+    # VPN
+    _ami_owners = ['099720109477']  # Amazon AMI owner
+    _vpn_ami_id_placeholder = "<ami_id>"
+    _vpn_ami_filters = [{'Name': 'virtualization-type', 'Values': ['hvm']},
+                        {'Name': 'architecture', 'Values': ['x86_64']},
+                        {'Name': 'name', 'Values': ['ubuntu/images/hvm-ssd/ubuntu-xenial*']}]
```

Comment:
The previous AMI `ami-49c9295f` is a trusty AMI. Has the VPN role been tested with xenial?